Java/samples/simple-console/src/main/java/com/microsoft/recognizers/text/samples/simpleconsole/Sample.java

```diff
@@ -17,7 +19,7 @@
 
 public class Sample {
 
-    private static final String defaultCulture = Culture.Spanish;
+    private static final String defaultCulture = Culture.English;
```

Comment:
Should these changes be part of this PR?

Reply:
This can be added in other PR, yes. I'll remove it

Reply:
Done